src/util/FactorUtil.js

```diff
@@ -64,71 +64,77 @@ function (Okta, TimeUtil) {
       iconClassName: 'mfa-google-auth',
       sortOrder: 5
     },
+    'CUSTOM_HOTP': {
+      label: '',
+      description: '',
+      iconClassName: 'mfa-custom-factor',
+      sortOrder: 6
```

Comment:
just double check with UX about the order 
cc @vincentvoong-okta (take a look at the test below to see the order of all factors)